### PR TITLE
Extend the update support to models with composite keys

### PR DESCRIPTION
**Pull request by gedgaudasnikita**
Currently update is only functional when applied to models with a single id column.

### CODE REVIEW

File: repositories/lib/services/entity.repository.js [hunk 1]
```diff
@@ -54,6 +54,15 @@ class EntityRepository {
   async update(entity, trx) {
     //Keep the input parameter immutable
     const entityDto = _.cloneDeep(entity);
+
+    const identityClause = {};
+
+    if (_.isArray(this.idColumn)) {
```
Comment (kibertoad) [inline repositories/lib/services/entity.repository.js]: considering that this just calls Array.isArray under the hood, using just that should be sufficient in modern Node :)

Comment (kibertoad) [inline repositories/lib/services/entity.repository.js]: Would you consider adding test for this case?

**Comment by kibertoad** [comment]
@gedgaudasnikita Glad to see a PR from you, old chap!

File: repositories/lib/services/entity.repository.js (hunk 1 above)
Comment (gedgaudasnikita) [reply repositories/lib/services/entity.repository.js]: I do. Gimme a few minutes, I'm working from a web GitHub IDE, just need to upload all the files from the local workstation 😅

**Review by kibertoad — APPROVED** [review]

**Comment by kibertoad** [comment]
@gedgaudasnikita Published as 1.8.0

How far ahead with TypeScript conversion are you guys, btw? I can convert this project to TypeScript if that would be helpful.

File: repositories/lib/services/entity.repository.js (hunk 1 above)
Comment (kibertoad) [reply repositories/lib/services/entity.repository.js]: Hardcore!

**Comment by gedgaudasnikita** [comment]
Thank you very much 🙏 .

In progress, @kibertoad  😄 We got a few services converted as a PoC, but there wasn't yet any need for typings on any Objection-related libraries. Though we might come back to this discussion soon 😏

**Pull request merged**